src/Response/Response.cpp

```diff
@@ -0,0 +1,94 @@
+#include "Response/Response.hpp"
+
+/*
+** ------------------------------- CONSTRUCTOR --------------------------------
+*/
+
+Response::Response() {}
```

Comment:
The default constructor could already fill the protocolVersion, since it will always have the same value.